manager/notifier.go

```diff
@@ -276,6 +278,46 @@ func (n *notifier) sendPushoverNotification(token, userKey string, a *Alert) err
 	return err
 }
 
+func (n *notifier) sendHipChatNotification(token, room string, a *Alert) error {
+	// https://www.hipchat.com/
+        incidentKey := a.Fingerprint()
+        buf, err := json.Marshal(map[string]interface{}{
```

Comment:
Hmm, now I get it. I thought this was constructing JSON for the API to consume. But it just becomes a message to be read by humans (incidentally in the PagerDuty JSON API format). Indeed, it should probably have a more human-readable content then.